v2/memoryv2_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package v2
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCgroupv2MemoryStats(t *testing.T) {
+	cg := NewCgroupDir("memory-test-cg", t)
+	defer clear(cg.groupPath)
+	res := Resources{
+		CPU:  &CPU{},
+		Pids: &Pids{},
+		IO:   &IO{},
+		RDMA: &RDMA{},
+		Memory: &Memory{
+			Max:  pointerInt64(629145600),
+			Swap: pointerInt64(314572800),
+			High: pointerInt64(524288000),
+		},
+	}
+	c, err := NewManager(cg.cgv2Root, "/memory-test-cg", &res)
+	if err != nil {
+		t.Fatal("failed to init new cgroup manager: ", err)
+	}
+	cg.writeFileContents(map[string]string{
```

Comment:
toggle the controller and write the limit files via the manager, and then verify the files via `ioutil.ReadFile`, and then also verify the stat struct from the manager.